libs/ui-watt/src/lib/components/text-field/watt-text-field.stories.ts

```diff
@@ -0,0 +1,163 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, moduleMetadata, StoryFn } from '@storybook/angular';
+import { FormControl, ReactiveFormsModule } from '@angular/forms';
+
+import { WattIconComponent } from '../../foundations/icon/icon.component';
+import { WattTextFieldComponent } from './watt-text-field.component';
+import { WattFieldComponent } from '../field';
+import { WattButtonComponent } from '../button';
+
+const meta: Meta<WattTextFieldComponent> = {
+  title: 'Components/Text Field',
```

Comment:
suggestion: looks like the documentation for the now "legacy" text field component is applied to this component. Maybe because the `storybook-input-overview.mdx` contains a Meta title with the same path as this component. Update the Meta title in that file to point to the legacy component instead.

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/2a757464-631c-4f1c-8ed3-35656ee07365)